static/build/miyoo/app/lang/nl.lang

```diff
@@ -0,0 +1,135 @@
+	"11":	"SMS",
+	"12":	"PS",
+	"13":	"PCE",
+	"14":	"OVERIG",
+	"15":	"Instellingen",
+	"16":	"GAME INFO",
+	"17":	"APPARAAT INFO",
+	"18":	"Recent",
+	"19":	"BESTAND",
+	"20":	"ADC INFO",
+	"21":	"EncDec Test",
+	"22":	"Knoppen opnieuw toewijzen",
+	"23":	"Taal wijzigen",
+	"24":	"Volume",
+	"25":	"Menugeluid",
+	"26":	"Helderheid",
+	"27":	"Roms vernieuwen",
+	"28":	"Fabrieksreset",
+	"29":	"Flitsmodus",
+	"30":	"Over het apparaat",
+	"31":	"Knoppen testen",
+	"32":	"ADC Info",
+	"33":	"Status:",
+	"34":	"Vertraging:",
+	"35":	"OK percentage:",
+	"36":	"Faalpercentage:",
+	"37":	"GESTOPT",
+	"38":	"ACTIEF",
+	"39":	"ADC:",
+	"40":	"Opladen",
+	"41":	"OTG",
+	"42":	"Verbroken",
+	"43":	"Fout",
+	"44":	"Toegevoegd aan favorieten",
+	"45":	"ANNULEREN",
+	"46":	"OK",
+	"47":	"seconden",
+	"48":	"in",
+	"49":	"Afspeellijst",
+	"50":	"Lijst wissen",
+	"51":	"Start",
+	"52":	"Verwijder uit lijst",
+	"53":	"Alle game caches leegmaken",
+	"54":	"Alle roms vernieuwen",
+	"55":	"Toevoegen aan favorieten",
+	"56":	"Bijwerken",
+	"57":	"Systeem bijwerken?",
+	"58":	"SD opslagruimte laag\n%dM over",
+	"59":	"Modelnummer",
+	"60":	"Serienummer",
+	"61":	"Firmwareversie",
+	"62":	"CPU-frequentie",
+	"63":	"%dMHz",
+	"64":	"Geheugengrootte",
+	"65":	"%dMB",
+	"66":	"Gebruikte/totaal opslag",
+	"67":	"Instellingen herstellen",
+	"68":	"Alle systeeminstellingen resetten?\nDe inhoud van de SD-kaart wordt\nniet verwijderd",
+	"69":	"Roms op SD-kaart vernieuwen",
+	"70":	"Herstellen",
+	"71":	"standaard",
+	"72":	"rotate90",
+	"73":	"map3",
+	"74":	"onbekend",
+	"75":	"LADEN",
+	"76":	"Bezig met laden",
+	"77":	"Recente lijst wissen?",
+	"78":	"Opmerking: SD-kaart inhoud\nwordt niet verwijderd.",
+	"79":	"Lijst met favorieten wissen?",
+	"78":	"Opmerking: SD-kaart inhoud\nwordt niet verwijderd.",
+	"81":	"Verwijderen uit favorieten lijst?",
+	"78":	"Opmerking: SD-kaart inhoud\nwordt niet verwijderd.",
+	"83":	"Scannen...",
+	"84":	"GAME INFO",
+	"85":	"Cache aanmaken",
+	"86":	"Afsluiten",
+	"87":	"Afsluiten over %d seconden...",
+	"88":	"SELECTEER",
+	"89":	"TERUG",
+	"90":	"NAV",
+	"91":	"MENU",
+	"92":	"Hervatten",
+	"93":	"Opslaan",
+	"94":	"Laden",
+	"95":	"Volume",
+	"96":	"Terug naar menu",
+	"97":	"State %d/%d",
+	"98":	"Geen state",
+	"99":	"Geen thumbnail",
+	"100":	"Laden mislukt",
+	"101":	"Opslaan mislukt",
+	"102":	"Slot %d opgeslagen",
+	"103":	"Volledig scherm",
+	"104":	"AAN",
+	"105":	"UIT",
+	"106":	"Geavanceerd",
+	"107":	"Apps",
+	"108":	"Weet u zeker dat je jouw apparaat\nuit wilt zetten?",
+	"109":	"Knoppen testen",
+	"110":	"Knoppen opnieuw toewijzen",
+	"111":	"AFSLUITEN",
+	"112":	"OPSLAAN EN AFSLUITEN",
+	"113":	"%d min",
+	"114":	"Uit",
+	"115":	"Slaaptimer",
+	"116":	"Game van SD-kaart verwijderen?",
+	"117":	"Bevestig dat je\ndeze game van je SD-kaart wilt verwijderen.",
+	"118":	"Weergave",
+	"119":	"Luminantie",
+	"120":	"Tint",
+	"121":	"Verzadiging",
+	"122":	"Contrast",
+	"123":	" \n \nWeergave-instellingen\nopgeslagen.\n \n ",
```

Comment:
Can you add a note, asking people to reboot the device (new for V4.1)

Reply:
Added reboot note.